projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas-header.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'go-off-canvas-header-actions',
+  template: `
+    <ng-template goPortalAttachTo="offCanvasHeader">
+      <ng-content></ng-content>
+    </ng-template>
+  `,
+  styles: []
```

Comment:
Should we remove this `styles` declaration if we're not assigning any styles to this component?

Reply:
yup. done